### PR TITLE
Added suggestions for production and exchange mp contracts.

### PR DESCRIPTION
**Pull request by erduna**
contracts.md will be updated in separate PR once consumption contract is also merged.

## References

* #0000

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ExchangeMeteringPointCreated.proto [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020 Energinet DataHub A/S
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ExchangeMeteringPointCreated.proto:1]: in #344 I've put Consumption contract in a folder /Consumption with the rest of the consumption related files.

Would suggest the same for event types.

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ProductionMeteringPointCreated.proto [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/* Copyright 2020 Energinet DataHub A/S
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ProductionMeteringPointCreated.proto:1]: same as above

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ExchangeMeteringPointCreated.proto (hunk 1 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ExchangeMeteringPointCreated.proto]: done

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ProductionMeteringPointCreated.proto (hunk 2 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/ProductionMeteringPointCreated.proto]: done

**Review by Korgath — DISMISSED** [review]

**Review by kft — DISMISSED** [review]

**Review by Korgath — APPROVED** [review]

**Pull request merged**